src/health-check/index.ts

```diff
@@ -1,10 +1,26 @@
 import assert from "assert";
-import { QuickAlert } from "../types";
+import { EventType, QuickAlert } from "../types";
+
+// CELO/cUSD rate feed address - only log health checks for this feed
+const CELO_CUSD_RATE_FEED_ADDRESS =
+  "0x765de816845861e75a25fca122bb6898b8b1282a".toLowerCase();
 
 export default function handleHealthCheckEvent(quickAlert: QuickAlert) {
   assert(
     quickAlert.blockNumber,
     "Block number is missing from MedianUpdated health check event",
   );
-  console.info("[HealthCheck]: Block", quickAlert.blockNumber);
+
+  // Only log health check for cUSD token
```

Comment:
is this just to reduce the amount of logs?

Reply:
yeah, if you remember we were seeing a ton of duplicate logs for the same healthcheck tx

the root cause was that imagine there was 1 block that had 7 TXs that triggered `MedianUpdated` events for 7 different feeds

previously, we'd have 1 block object for this with 7 matching logs for `MedianUpdated` which led to the health check being printed 7 times

now it should be only once